pkg/registry/networking/ingress/strategy_test.go

```diff
@@ -138,3 +141,62 @@ func TestIngressStatusStrategy(t *testing.T) {
 		t.Errorf("Unexpected error %v", errs)
 	}
 }
+
+func TestWarningsOnCreate(t *testing.T) {
+	ctx := genericapirequest.NewDefaultContext()
```

Comment:
Why are these in this test?  I don't think we need them, much less in this unrelated test.

Reply:
Because we changed `WarningsOnCreate` in `strategy.go`, so I created a test in `strategy_test.go`. 
Removed in new commit.